src/prognostic_equations/edmfx_sgs_flux.jl

```diff
@@ -302,10 +305,10 @@ function edmfx_sgs_mass_flux_tendency!(
                         )
                     vtt = vertical_transport(
                         ᶜρʲs.:($j),
-                        ᶠu³ʲs.:($j),
+                        ᶠu³_diff,
                         ᶜa_scalar,
                         dt,
-                        edmfx_tracer_upwinding,
+                        edmfx_sgsflux_upwinding,
```

Comment:
You don't need to change this. `edmfx_sgsflux_upwinding` is None by default and it was better for 0m previously, but I didn't test it for 1M.